### PR TITLE
added Fotia gorget and belt to ws mods

### PR DESCRIPTION
**Pull request by dwn134**

### CODE REVIEW

File: scripts/globals/weaponskills.lua [hunk 1]
```diff
@@ -1043,6 +1048,11 @@ function handleWSGorgetBelt(attacker)
                 break;
             end
         end
+
+        if (belt == 28420) then
+                accBonus = accBonus + 10;
+                ftpBonus = ftpBonus + 0.1;
```
Comment (takhlaq) [inline scripts/globals/weaponskills.lua]: 4 space indent

File: scripts/globals/weaponskills.lua [hunk 2]
```diff
@@ -1034,6 +1034,11 @@ function handleWSGorgetBelt(attacker)
             end
         end
 
+        if (neck == 27510) then
+        	    accBonus = accBonus + 10;
```
Comment (takhlaq) [inline scripts/globals/weaponskills.lua]: tabs > 4 spaces

File: scripts/globals/weaponskills.lua [hunk 3]
```diff
@@ -1034,6 +1034,11 @@ function handleWSGorgetBelt(attacker)
...
+                ftpBonus = ftpBonus + 0.1;
```
Comment (takhlaq) [inline scripts/globals/weaponskills.lua]: 4 space indent

File: scripts/globals/weaponskills.lua [hunk 4]
```diff
@@ -1043,6 +1048,11 @@ function handleWSGorgetBelt(attacker)
                 break;
             end
         end
+
+        if (belt == 28420) then
```
Comment (takhlaq) [inline scripts/globals/weaponskills.lua]: not a fan of using ids but guess there isnt really a better way
a comment with which item it is would be nice though, for anyone who needs to look up how this is handled later

**Comment by wrenffxi** [comment]
If this is coded as an equipment slot check rather than an item mod + mod check, then players will receive the bonus when wearing the item even when under level sync.

**Comment by dwn134** [comment]
It coded as all the other gorgets right above though

**Comment by TeoTwawki** [comment]
It's yet another case of person before you got here being naughty.

@takhlaq why you no teach'em how to modifier? 😢

**Comment by Hozu** [comment]
It was like this before I moved the gorget/belt check into its own function so it could be called upon for Atonement. The problem I couldn't figure out at the time was how to avoid doubling up on the bonus on certain WSs that had SC properties that could both count for a gorget, like every single relic WS.

...but thinking about it now it could probably do a for loop that runs for as long as many properties the WS has, comparing each individual gorget mod, and using a break to exit early once it's met. Nested inside another for loop for the 8 gorget elements. And before that a check for the all element gorget mod. Unless someone can figure out a way for it to work with all 8 modifiers.

**Comment by TeoTwawki** [comment]
@dwn134 now you got multiple diff things smushed into one branch. This is why you should make new branches for each thing you work on. 

You'll have to rebase ***do not simply close pr and or delete your entire fork to start over.*** 

Make a new branch to rebase to darkstar master and cherrypick the commits that didn't belong here into that, then rebase or reset this one to eliminate them. If you can't figure it out **ask in discord for help.**

**Comment by dwn134** [comment]
Yea. I hit the new pull request button and I didn't know it would still merge to my last pull

**Comment by TeoTwawki** [comment]
it didn't, git is based on branches. you worked in same branch for all these commits.

**Comment by dwn134** [comment]
fixed

File: scripts/globals/weaponskills.lua [hunk 5]
```diff
@@ -1034,6 +1034,11 @@ function handleWSGorgetBelt(attacker)
             end
         end
 
+        if (neck == 27510) then -- Fotia Gorget
+                accBonus = accBonus + 10;
+                ftpBonus = ftpBonus + 0.1;
```
Comment (takhlaq) [inline scripts/globals/weaponskills.lua]: 4 space indent please

**Comment by dwn134** [comment]
Ah I had that fixed. Whatever I did took it back to the beginning. ><

**Pull request merged**